Pytest-redis 3.1.0 factories are broken

### What action do you want to perform

```py
from pytest_redis import factories

redis_proc = factories.redis_proc(host="127.0.0.1", port=26379)
```

### What are the results

```py
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'pytest_redis.factories' has no attribute 'redis_proc'
```

### What are the expected results

I'd like to import the `redis_proc` from `factories` successfully just like in the version 3.0.2

cc @fizyk

We are getting errors in our unit tests
```
E           mirakuru.exceptions.ProcessExitedWithError: The process invoked by the <pytest_redis.executor.process.RedisExecutor: "/usr/bin/redis-server --daemonize no --rdbcompression no --rdbchecksum no --appendonly no --databases 8 --timeout 30 --pidfile /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis-server.31926.pid --unixsocket /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis.31926.sock --dbfilename dump.31926.rdb --logfile /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis-server.31926.log --loglevel notice --syslog-enabled no --bind 127.0.0.1 --port 31926 --dir /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0 --loadmodule ''" 0x7f3fdfd0fdf0> executor has exited with a non-zero code: 1.```

--- a/newsfragments/679.bugfix.rst
+++ b/newsfragments/679.bugfix.rst
@@ -1,0 +1,2 @@
+Fixed redis factories module, where imports from submodule
+on the main `factories` level were removed by linter.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -68,7 +68,7 @@ include = '.*\.pyi?$'
 [tool.ruff]
 # Decrease the maximum line length to 79 characters.
 line-length = 100
-select = [
+lint.select = [
     "E",   # pycodestyle
     "F",   # pyflakes
     "I",   # isort

--- a/pytest_redis/factories/__init__.py
+++ b/pytest_redis/factories/__init__.py
@@ -1,1 +1,7 @@
 """Redis fixture factories."""
+
+from pytest_redis.factories.client import redisdb
+from pytest_redis.factories.noproc import redis_noproc
+from pytest_redis.factories.proc import redis_proc
+
+__all__ = ("redis_proc", "redis_noproc", "redisdb")

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -2,9 +2,7 @@
 
 import warnings
 
-import pytest_redis.factories.client
-import pytest_redis.factories.noproc
-import pytest_redis.factories.proc
+import pytest_redis.factories
 from pytest_redis.plugin import *  # noqa: F403
 
 warnings.filterwarnings(
@@ -12,13 +10,13 @@ warnings.filterwarnings(
 )
 
 # pylint:disable=invalid-name
-redis_proc2 = pytest_redis.factories.proc.redis_proc(port=6381)
-redis_nooproc2 = pytest_redis.factories.noproc.redis_noproc(port=6381, startup_timeout=1)
-redis_proc3 = pytest_redis.factories.proc.redis_proc(port=6385, password="secretpassword")
-redis_nooproc3 = pytest_redis.factories.noproc.redis_noproc(port=6385, password="secretpassword")
+redis_proc2 = pytest_redis.factories.redis_proc(port=6381)
+redis_nooproc2 = pytest_redis.factories.redis_noproc(port=6381, startup_timeout=1)
+redis_proc3 = pytest_redis.factories.redis_proc(port=6385, password="secretpassword")
+redis_nooproc3 = pytest_redis.factories.redis_noproc(port=6385, password="secretpassword")
 
-redisdb2 = pytest_redis.factories.client.redisdb("redis_proc2")
-redisdb2_noop = pytest_redis.factories.client.redisdb("redis_nooproc2")
-redisdb3 = pytest_redis.factories.client.redisdb("redis_proc3")
-redisdb3_noop = pytest_redis.factories.client.redisdb("redis_nooproc3")
+redisdb2 = pytest_redis.factories.redisdb("redis_proc2")
+redisdb2_noop = pytest_redis.factories.redisdb("redis_nooproc2")
+redisdb3 = pytest_redis.factories.redisdb("redis_proc3")
+redisdb3_noop = pytest_redis.factories.redisdb("redis_nooproc3")
 # pylint:enable=invalid-name

